controllers/chat/deleteMessage.js

```diff
@@ -0,0 +1,31 @@
+const {StreamChat} = require('stream-chat');
+
+const deleteMessage = async (req, res) => {
+  const client = new StreamChat(process.env.API_KEY, process.env.SECRET_KEY);
+  const {messageID} = req.params;
+
+  try {
+    await client.connectUser(
+      {
+        id: req.userId
+      },
+      req.token
+    );
```

Comment:
I think we don't need connect user for this.
Just use client instance as is.

Reply:
Get error if not connect user